### PR TITLE
Move to PSR7-session organization

### PR DESCRIPTION
**Pull request by Ocramius**
This is what we need to get to 3.0.0.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,32 +1,6 @@
 This is a list of changes/improvements that were introduced in PSR7Session
 
-## 2.0.0
+## 1.0.0
```
Comment (lcobucci) [inline CHANGELOG.md]: We should review these changes

File: docs/configuration.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ $sessionMiddleware = new SessionMiddleware(
     $cookieBlueprint,
     new \Lcobucci\JWT\Parser(),
     1200, // session lifetime, in seconds
-    new \PSR7Session\Time\SystemCurrentTime(), // Current time provider implementation using current time system
+    new \PSR7Session\Storageless\Time\SystemCurrentTime(), // Current time provider implementation using current time system
```
Comment (lcobucci) [inline docs/configuration.md]: Just noticed: `s/current time system/current system time`

File: composer.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-    "name":     "ocramius/psr7-session",
+    "name":     "psr7-sessions/storageless",
```
Comment (lcobucci) [inline composer.json]: - `description` is missing
- `replace` should be added

File: README.md [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
-[![Packagist](https://img.shields.io/packagist/v/ocramius/psr7-session.svg)](https://packagist.org/packages/ocramius/psr7-session)
-[![Packagist](https://img.shields.io/packagist/vpre/ocramius/psr7-session.svg)](https://packagist.org/packages/ocramius/psr7-session)
+[![Build Status](https://travis-ci.org/psr7-sessions/storageless.svg)](https://travis-ci.org/psr7-sessions/storageless)
+[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/psr7-sessions/storageless/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/psr7-sessions/storageless/?branch=master)
+[![Code Coverage](https://scrutinizer-ci.com/g/psr7-sessions/storageless/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/psr7-sessions/storageless/?branch=master)
+[![Packagist](https://img.shields.io/packagist/v/psr7-sessions/storageless.svg)](https://packagist.org/packages/psr7-sessions/storageless)
+[![Packagist](https://img.shields.io/packagist/vpre/psr7-sessions/storageless.svg)](https://packagist.org/packages/psr7-sessions/storageless)
 
 **PSR7Session** is a [PSR-7](http://www.php-fig.org/psr/psr-7/)
```
Comment (lcobucci) [inline README.md]: Maybe mention that the storageless package is not only about session usage in PSR-7 based applications but also without storing anything (it sounds a bit stupid when put like this but is just to differentiate from the other possible packages of the org)?

**Comment by danizord** [comment]
OCD alert: The Github organization name is in plural while the vendor namespace is not. :P

**Comment by Ocramius** [comment]
We'll have multiple session implementations here ;-)

**Comment by danizord** [comment]
>We'll have multiple session implementations here ;-)

So the vendor namespace should be in plural as well?

**Comment by Ocramius** [comment]
Oh, I see what you mean. `PSR7Sessions\Storageless` then?

Marco Pivetta

http://twitter.com/Ocramius

http://ocramius.github.com/

**Comment by danizord** [comment]
>Oh, I see what you mean. `PSR7Sessions\Storageless` then?

Yup

**Comment by malukenho** [comment]
@Ocramius @danizord done

File: composer.json [hunk 5]
```diff
@@ -27,16 +28,17 @@
...
     "require-dev": {
         "phpunit/phpunit":              "^5.0",
-        "zendframework/zend-diactoros": "^1.1"
+        "zendframework/zend-diactoros": "^1.1",
+        "humbug/humbug":                "dev-master"
     },
     "autoload": {
         "psr-4": {
-            "PSR7Session\\": "src/PSR7Session"
+            "PSR7Sessions\\Storageless\\": "src/Storageless"
```
Comment (danizord) [inline composer.json]: If we're trying to mimic PSR-0 here, then the path should `src/PSR7Sessions/Storageless`. Otherwise nvm :)

**Comment by Ocramius** [comment]
Besides review, this seems good to go. Anyone want to think about switching to 7.1, eventually?

@lcobucci can you do a final review?

**Comment by lcobucci** [comment]
> Anyone want to think about switching to 7.1, eventually?

I kind of like the idea but since the library is really small we could decide this closer the release of v3 (we still need to extract some code from here in order to have the other types of session), probably we'll have a stable PHP 7.1.

**Comment by Ocramius** [comment]
@lcobucci V3 is complete, just needs release notes/releasing/signed tag.

**Comment by lcobucci** [comment]
Alright then so let's discuss that for v4.

**Comment by Ocramius** [comment]
:+1:

Marco Pivetta

http://twitter.com/Ocramius

http://ocramius.github.com/

**Pull request merged**